app/src/main/java/de/westnordost/streetcomplete/quests/accepts_cash/AddAcceptsCash.kt

```diff
@@ -0,0 +1,55 @@
+package de.westnordost.streetcomplete.quests.accepts_cash
+
+import de.westnordost.streetcomplete.R
+import de.westnordost.streetcomplete.data.osm.NoCountriesExcept
+import de.westnordost.streetcomplete.data.osm.SimpleOverpassQuestType
+import de.westnordost.streetcomplete.data.osm.changes.StringMapChangesBuilder
+import de.westnordost.streetcomplete.data.osm.download.OverpassMapDataAndGeometryDao
+import de.westnordost.streetcomplete.quests.YesNoQuestAnswerFragment
+
+class AddAcceptsCash(o: OverpassMapDataAndGeometryDao) : SimpleOverpassQuestType<Boolean>(o) {
+//nodes, ways with amenity = toilets and access !~ private|customers and !fee
+    private val amenity = listOf(
+        "bar", "cafe", "fast_food", "food_court", "ice_cream", "pub", "biergarten",
+        "restaurant", "cinema", "nightclub", "planetarium", "theatre", "marketplace",
+        "internet_cafe", "car_wash", "fuel", "pharmacy"
+    )
+    private val tourism = listOf(
+        "zoo", "aquarium", "theme_park", "museum"
+    )
+    private val leisure = listOf(
+        "tanning_salon"
+    )
+    private val craft = listOf(
+        "carpenter", "shoemaker", "tailor", "photographer", "dressmaker",
+        "electronics_repair", "key_cutter", "stonemason"
+    )
+
+    override val tagFilters = """
+        nodes, ways, relations with
+        (
+          (shop and shop !~ no|vacant|mall)
+          or amenity ~ ${amenity.joinToString("|")}
+          or leisure ~ ${leisure.joinToString("|")}
+          or craft ~ ${craft.joinToString("|")}
+          or tourism ~ ${tourism.joinToString("|")}
+        )
+        and name and !payment:cash and !payment:coins and !payment:notes
+    """
```

Comment:
I think it would be better readable if you put the values directly into the tag filter string like you did for the shops. But this is not blocking a merge

Reply:
I avoided long lines with foo|bar|baz|… I'll leave it like so.